Source/Engine/Level/Actor.cpp

```diff
@@ -1806,3 +1808,26 @@ void Actor::FromJson(const StringAnsiView& json)
     Scripting::ObjectsLookupIdMapping.Set(nullptr);
     OnTransformChanged();
 }
+
+void Actor::SetPhysicsScene(PhysicsScene* scene)
+{
+    ASSERT(scene);
+
+    auto previous = GetPhysicsScene();
+    _physicsScene = scene;
+
+    if (previous != _physicsScene)
+        OnPhysicsSceneChanged(previous);
+
+    // cascade
+    for (auto child : Children)
```

Comment:
Call `SetPhysicsScene` for children only if physics scene gets changed for this actor.

Reply:
Done